assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -74,6 +74,7 @@ function AssignmentPlanningV2 (props) {
 
   const [assignments, setAssignments] = useState([])
   const [goalGrade, setGoalGrade] = useState('')
+  const [goalGradePrev, setGoalGradePrev] = useState('')
```

Comment:
> When I set an assignment grade, the previous course goal grade is getting logged as 0

I did set an empty string but internally when useState is called it is setting to 0. or probably internally goalGrade is zero so it is like that

`const preGrade = usePrevious(goalGrade)`

Is that confusing? should we change it to empty string ''

Reply:
I think ideally the goalGradePrev property wouldn't be present unless there was a change... what do you think?

Reply:
Ok, I am only sending the `gradePrev` value when it is there

Reply:
I just updated my course goal from grade  from 86 to 84 and the previous  grade (86) was not in the log

```
{
   "course_id":17700000000400707,
   "default_type":"assignment",
   "default_value":{
      "goalGrade":"84",
      "currentGrade":91.6,
      "maxPossible":94.6,
      "assignments":[
```

Reply:
It is at the very end of the object? can you check?